### DISCUSSION TITLE
Duplicate Project

### DISCUSSION THREAD

**Discussion by root9191** [Leantime/leantime#866] [Q&A]
Hello, 
so in every Milestone i had some Information as Discussion configured and when i duplicate a Project my Discussions are gone. 
Could i create a Template with all my Informations in every Milestone and duplicate all of this?

**Comment by marcelfolaron** [comment]
Hi,

Thanks for reaching out.
The system does not currently copy discussions as we felt like this was
always something specific to the project. Can I ask why that information is
in discussions and not in the description of the task? Just trying to
understand the use case.

**Reply by root9191** [reply]
OK that's why :)
In my Projects one Milestone is one Week and in every Week i wrote some Information i need for this Week. 
If i use the Description in the Task i do not have that Overview in the Milestone Tab.

**Comment by marcelfolaron** [comment]
I see, thanks. Now in this case wouldn’t it be better to have a description
field on the milestone? I know it’s not there right now, but that would be
a fairly simple thing to add.

**Reply by root9191** [reply]
That would be cool, but i think it's no Problem for me to write it in the Description of the Task.
